src/main/interface/cli.c

```diff
@@ -970,13 +998,21 @@ static void cliSerialPassthrough(char *cmdline)
                    passThroughPort->mode, mode);
             serialSetMode(passThroughPort, mode);
         }
+
+        // Set the baud rate
+        // Needed as arduino bootloaders may run at 57600 baud
+        serialSetBaudRate(passThroughPort, baud);
```

Comment:
This should be called only if user supplied new baudrate

Reply:
Noted. Fixed.